data_model/src/lib.rs

```diff
@@ -712,16 +712,7 @@ where
 /// and `PartialCmp` implementations.
 pub trait Identifiable: Debug {
     /// The type of the `Id` of the entity.
-    type Id: Into<IdBox>
-        + fmt::Display
-        + fmt::Debug
-        + Clone
-        + Eq
-        + Ord
-        + core::hash::Hash
-        + Decode
-        + Encode
-        + Serialize;
+    type Id;
```

Comment:
With https://github.com/hyperledger/iroha/pull/2497#discussion_r923344278 I've changed my heart. These bounds on `Id` should not be involved in the `Identifiable` trait declaration